apps/web/app/[locale]/auth/passcode/page.tsx

```diff
@@ -101,10 +101,13 @@ function EmailScreen({ form, className }: { form: TAuthenticationPasscode } & IC
 
 					<div className="flex items-center justify-between w-full mt-6">
 						{/* Send code */}
-						<div className="flex flex-col items-start">
-							<Link href="/auth/team">
-								<BackButton />
-							</Link>
+						<div className="flex flex-col items-start gap-2">
+							<div className="flex justify-start items-center text-sm">
+								<span>{t('common.BACK')}</span>
```

Comment:
@Cedric921 can you pay attention to small details? E.g. common.BACK is NOT "Back" anymore, it's `common.DONT_HAVE_ACCOUNT` or something like that...